### DISCUSSION TITLE
Fun with Lesson 4: Ganache TX hash looks like a curse b'/S@"\xf9\x90\xab\xacG\xfd\xb2\x08\x ...

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#922] [General]
If (like me) you sometimes get curious about what's in the parameters passed to a function, you may have noticed in Lesson 4 that tx_hash looks strange. Example:
`b'/S@"\xf9\x90\xab\xacG\xfd\xb2\x08\x95\xae\xf5\xca\xbb\xb3\x85ys\xf4\x19\x83\x89\xe4\xe2\xcf\xf5^\xb7$'`
This is because [`Eth.send_raw_transaction`](https://web3py.readthedocs.io/en/stable/web3.eth.html?highlight=send_raw_transaction#web3.eth.Eth.send_raw_transaction) returns the transaction hash as a **HexBytes** object. 
If you look at Ganache under Transactions, you won't find this TX hash. An easy way to get the hash in hex format is to use `print("tx_hash: " + str(tx_hash.hex()))` in your code.

**Comment by cromewar** [comment]
Awesome @n4n0b1t3 you are helping the community a lot!
